examples/Cassie/cassie_utils.cc

```diff
@@ -126,6 +126,65 @@ void buildCassieTree(RigidBodyTree<double>& tree, std::string filename,
   ankle_spring_joint_right.SetSpringDynamics(1250.0, 0.0);  // 2300 in URDF
 }
 
+drake::systems::sensors::Accelerometer * addSimAccelerometer(
```

Comment:
Right. I'm passing the frame name (string) instead of RigidBodyFrame now.

Reply:
Why would the user specify the frame name? That's true for some general robot, but this utilty is Cassie-specific.

Reply:
I was thinking that since the frame name is declared in run_simple_sim (https://github.com/DAIRLab/dairlib/blob/2e1d1f64029a194c260cd43f22c7ff4f97f96c53/examples/Cassie/run_simple_sim.cc#L70),  I should pass in the name when I call the function. 

But on second thought, I agree that users shouldn't touch the frame name. I hid the name in cassie_utils just now.